lib/models/ente_file.dart

```diff
@@ -1,14 +1,12 @@
 // EnteFile is base file entry for various type of files
 // like DeviceFile,RemoteFile or TrashedFile
 
-// @dart=2.9
-
 abstract class EnteFile {
   // returns cacheKey which should be used while caching entry related to
   // this file.
-  String cacheKey();
+  String? cacheKey();
```

Comment:
When will the `cacheKey` be null?

Reply:
@ua741 Have answered this on a comment on `file.dart`